gateway/ln-gateway/src/lightning/mod.rs

```diff
@@ -111,6 +118,35 @@ pub trait ILnRpcClient: Debug + Send + Sync {
         &self,
         htlc: InterceptHtlcResponse,
     ) -> Result<EmptyResponse, LightningRpcError>;
+
+    /// Connect to a peer lightning node from the gateway's lightning node.
+    async fn connect_to_peer(
+        &self,
+        pubkey: secp256k1::PublicKey,
+        host: String,
+    ) -> Result<EmptyResponse, LightningRpcError>;
+
+    /// Get the funding address belonging to the gateway's lightning node
+    /// wallet.
+    async fn get_funding_address(&self) -> Result<GetFundingAddressResponse, LightningRpcError>;
+
+    /// Open a channel with a peer lightning node from the gateway's lightning
+    /// node.
+    async fn open_channel(
+        &self,
+        pubkey: String,
```

Comment:
Can this be a `secp256k1::PublicKey` too?

Reply:
Done.